### PR TITLE
xmrig: 2.14.4 -> 3.0.0

### PR DESCRIPTION
**Pull request by kim0**
###### Motivation for this change
Upgrade xmrig 2.14.4 -> 3.0.0

###### Things done
added hwloc dependency, and bumped version

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @fpletz

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2745,6 +2745,11 @@
     github = "kiloreux";
     name = "Kiloreux Emperex";
   };
+  kim0 = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Do you mind splitting this into a separate commit?

**Comment by mmahut** [comment]
@GrahamcOfBorg build xmrig

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (kim0) [reply maintainers/maintainer-list.nix]: Done, thanks @mmahut

**Comment by kim0** [comment]
@mmahut .. Morning, is it acceptable now ? Thanks

**Comment by mmahut** [comment]
@GrahamcOfBorg build xmrig
